backend/models/room.py

```diff
@@ -22,7 +22,7 @@ class Room(db.Model):
     seats = db.relationship(
         'Seat',
         back_populates="room",
-        foreign_keys=[Seat.room_id, Seat.seat_id],
+        foreign_keys=[Seat.room_id, Seat.id],
```

Comment:
I think you can remove the foreign_keys-field. Look at how it's done in other relations